### PR TITLE
Adding post on Slack and Voluntary Adoption.

### PR DESCRIPTION
**Pull request by trenaman**
Put together this post on how we applied to principal of voluntary adoption in our move to Slack.

### CODE REVIEW

**Comment by trenaman** [comment]
@johncoghlan @JustinRZA @jvitale9 @heatherfleming interested in your thoughts.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: On Slack: Musings on Decentralised Control and Voluntary Adoption in Large Organisations.
+author: Adrian Trenaman 
+date: '2017-02-02'
+categories: 
+tags:
+- leadership 
+---
+
+
+When I think of Slack, I think first of the great book by Tom DeMarco on the organisational slack we need to play, innovate, get big things done. It’s an amazing read, and I recommend without reservation. More recently, when I think of Slack, I think of the massive grassroots movement at HBC Digital that switched us from HipChat to Slack in just a few short weeks, without any top-down edict or stop-the-world migration.  We have this core value in Tech called ‘voluntary adoption’: if a technology, framework, tool or service is really good, then your teams will adopt it naturally, without coercion. The corollary of voluntary adoption is that if you find that you’re having to push a solution on a group of people and they’re resisting, pushing back, or not getting it, then it’s a good sign that the solution might not be as good as you previously thought.
```
Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Help readers interested in the book by adding a link. https://www.amazon.com/Slack-Getting-Burnout-Busywork-Efficiency/dp/0767907698

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: On Slack: Musings on Decentralised Control and Voluntary Adoption in Large Organisations.
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: title is far too wordy for my tastes, will look a bit funky in a rendered layout, and consumes 90 characters in a tweet. I'd shorten that to 5 or so words and move the remaining to a brief subtitle or quoted section at the beginning of the body.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+author: Adrian Trenaman 
+date: '2017-02-02'
+categories:
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: should have at least one category

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: On Slack: Musings on Decentralised Control and Voluntary Adoption in Large Organisations.
+author: Adrian Trenaman 
+date: '2017-02-02'
+categories: 
+tags:
+- leadership 
+---
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: let's add some more tags. that'll aid the googles

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 1 above)
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: slack vs. slack - confusing at first glance, perhaps some more distinction between the app and the concept of slack is needed.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+We apply voluntary adoption at multiple levels - in our open-source offerings, internal libraries, tools, and, how we work together as teams. And the ramifications for voluntary adoption for us as engineers and product innovators are profound. If you’re going to invest time into building something, spend time on making it a dream for the newcomer: easy to use, surprising, delighting. Think: you are not in a position to force your solution on someone: but you can make your solution something that’s a dream to adopt. Voluntarily.
```
Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: There's an extra comma after "and" in the first sentence but it might be there for effect.

Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Might go with this: We apply voluntary adoption at multiple levels - in our open-source offerings, internal libraries, tools, and how we work together as teams - and the ramifications for voluntary adoption for us as engineers and product innovators are profound.

Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: The second colon in the last sentence seems unnecessary.

Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Might go with this: We apply voluntary adoption at multiple levels - in our open-source offerings, internal libraries, tools, and how we work together as teams - and the ramifications for voluntary adoption for us as engineers and product innovators are profound.

Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: The second colon in the last sentence seems unnecessary.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+Through merger and acquisition, we found ourselves in a position with multiple tech teams using different chat solutions, creating artificial divisions and cross-team awkwardness. We could have mandated a move to one of the incumbent chat solutions at HBC and dragged everyone across the divide, a solution that would have been a long hard march. Instead, we looked about at the current most-loved tool, Slack, and kicked off a couple of channels and invited some folk in, and said, ‘hey, try it out.’ Within days, we encountered some interesting effects: first, folk loved it; and second, folk wanted clarity to know if everyone could just move there together. Without having to force or coerce anyone, we’re now all together on one system, Slack.
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: "divide, a solution" > "divide - a solution"
"folk in, and said" unneeded comma
‘hey, try it out.’ - always preferred double quotes because oxford
"Within days, we" unneeded comma
"interesting effects: first, folk loved it; and second, folk" > "interesting effects; firstly folk loved it, and second, folk"
"on one system, Slack" - semicolon

may be a queen's english vs american: folk, or folks?

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+So what do we learn from this application of voluntary adoption? First, we got the outcome we wanted, fast, and it stuck. Second, but perhaps more interestingly, was that we traded off one kind of organisational stress for another. Top down, authoritative control offers clarity and a sense of control, and the expense of individual choice. “Everyone, we’re using Tool X’ has a clarity, but folk quickly reject being told to use a tool they don’t like and that causes stress and angst. “Everyone, we don’t have an agreed standard yet so why not try this as well as the current solutions?” feels rudderless and perhaps somewhat chaotic for those in the midst of it: adoptees are confused and wonder which one to choose. However, this approach unleashes a Darwinian self-selection, a decentralised, collaborative optimisation process that will either squash a bad idea up front or elevate a good idea into something grand.
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: “Everyone, we’re using Tool X’ - mixing quote styles
" like and that causes" > " like and that which causes"

"Darwinian self-selection" I'd link to an explanation. Freely admitting I have no clue what this means.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 5 above)
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: "levels - in" mayyyybe a semicolon, but a hyphen works.
"for us" superfluous 
"Think:" def a semi
"someone: but" > "someone, but"

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+We apply voluntary adoption at multiple levels - in our open-source offerings, internal libraries, tools, and, how we work together as teams. And the ramifications for voluntary adoption for us as engineers and product innovators are profound. If you’re going to invest time into building something, spend time on making it a dream for the newcomer: easy to use, surprising, delighting. Think: you are not in a position to force your solution on someone: but you can make your solution something that’s a dream to adopt. Voluntarily. 
+
+
```
Comment (shellscape) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: recommend setting editor to trim extra newlines

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 7 above)
Comment (ahollinshead) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Not sure about "folk quickly reject being told to use a tool they don’t like and that which causes stress and angst" - doesn't seem to scan properly. It may depend on whether it's the tool that causes stress and angst or the being told to use the tool.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 5 above)
Comment (ahollinshead) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Strictly speaking should it be an em dash not a hyphen?

Comment (ahollinshead) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Maybe 'delighting' -> 'delightful'. Delight without an object I think means the solution is becoming very happy. Alternative 'able to delight'?

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 3 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Agree.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 2 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: how about 'Voluntary Adoption in Action: HBC Digital Adopts Slack'

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 4 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: done.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 1 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Changing line width. Correct use of colon; semi-colon is not appropriate there! Using Pronoun Slack now to differntiate between "slack", which I'm now putting in place.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 6 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: Changing to use colon instead of '-'
Agree on superfluous comma
Agree on quotes.
I stand over my use of colons and semi-colons!

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 5 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: done

File: _posts/2017-02-02-slack-and-voluntary-adoption.md (hunk 7 above)
Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: changed.

File: _posts/2017-02-02-slack-and-voluntary-adoption.md [hunk 9]
```diff
@@ -1,20 +1,65 @@
...
+a group of people and they’re resisting, pushing back, or not getting it, then
+it’s a good sign that the solution might not be as good as you previously
+thought.
 
-We apply voluntary adoption at multiple levels - in our open-source offerings, internal libraries, tools, and, how we work together as teams. And the ramifications for voluntary adoption for us as engineers and product innovators are profound. If you’re going to invest time into building something, spend time on making it a dream for the newcomer: easy to use, surprising, delighting. Think: you are not in a position to force your solution on someone: but you can make your solution something that’s a dream to adopt. Voluntarily. 
+Through merger and acquisition, we found ourselves in a position with multiple
+tech teams using different chat solutions, creating artificial divisions and
+cross-team awkwardness. We could have mandated a move to one of the incumbent
+chat solutions at HBC and dragged everyone across the divide: a solution that
+would have been a long hard march. Instead, we looked about at the current most-
```
Comment (johncoghlan) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: There is an unnecessary space after the hyphen in "most-loved".  Other than that, looks good to merge.

Comment (trenaman) [inline _posts/2017-02-02-slack-and-voluntary-adoption.md]: done
